config/express.js

```diff
@@ -0,0 +1,122 @@
+const log = require('../app/lib/logger');
```

Comment:
Can the requires be organised alphabetically and with external dependencies listed before internal modules please?